Multisig/Logic/Models/Safe.swift

```diff
@@ -201,10 +211,14 @@ extension Safe: Identifiable {
 
 extension Safe {
     func update(from info: SafeInfoRequest.ResponseType) {
+        DispatchQueue.main.async {
```

Comment:
why is this async? I thought just saving the object after assigning `contractVersion` would work

Reply:
Update might happen on background thread.